docs/reference/events.rst

```diff
@@ -79,11 +65,25 @@ about--``ScenePaused.scene`` is the scene that is being paused.
 .. autoclass:: ppb.events.SceneStarted
```

Comment:
https://bluesock.org/~willkg/blog/dev/emdash_in_sphinx.html

Because apparently `---` is an em dash. Also, I don't know if I'm the one who wrote that.

Reply:
It was probably me because I think I last touched this page.